validphys2/src/validphys/coredata.py

```diff
@@ -370,17 +385,31 @@ def systematic_errors(self, central_values=None):
         converted_mult_errors = self.multiplicative_errors * central_values[:, np.newaxis] / 100
         return pd.concat((self.additive_errors, converted_mult_errors), axis=1)
 
-    def export(self, path):
-        """Export the data, and error types
-         Use the same format as libNNPDF:
-
-        - A DATA_<dataset>.dat file with the dataframe of accepted points
-        - A systypes/STYPES_<dataset>.dat file with the error types
-        """
-
-        dat_path = path / f"DATA_{self.setname}.dat"
-        sys_path = path / "systypes" / f"SYSTYPE_{self.setname}_DEFAULT.dat"
-        sys_path.parent.mkdir(exist_ok=True)
-
-        write_systype_to_file(self, sys_path)
-        write_commondata_to_file(self, dat_path)
+    def export_data(self, buffer):
```

Comment:
Since `commondata.export` has been removed, its previous occurrences need to be adjusted as well. Here for example (?):
https://github.com/NNPDF/nnpdf/blob/6d05ace3aa4aa421fecb4702f92cbe6ced0b412a/validphys2/src/validphys/filters.py#L214